poly-commitment/src/lib.rs

```diff
@@ -60,13 +63,24 @@ pub trait SRS<G: CommitmentCurve>: Clone + Sized {
     }
 
     /// This function commits a polynomial using the SRS' basis of size `n`.
-    /// - `plnm`: polynomial to commit to with max size of sections
-    /// - `num_chunks`: the number of commitments to be included in the output
-    /// polynomial commitment
-    /// The function returns an unbounded commitment vector (which splits the
-    /// commitment into several commitments of size at most `n`).
-    /// It is analogous to [SRS::commit_evaluations_non_hiding] but for
-    /// polynomials.
+    /// - `plnm`: polynomial to commit to. The polynomial can be of any degree,
+    /// including higher than `n`.
+    /// - `num_chunks`: the minimal number of commitments to be included in the
+    /// output polynomial commitment.
+    ///
+    /// The function returns the commitments to the chunks (of at most `n`) of
```

Comment:
I guess there's a missing word in that sentence. Didn't you mean? 
```suggestion
    /// The function returns the commitments to the chunks (of size at most `n`) of
```